### DISCUSSION TITLE
Compare bias-corrected footprinting values between groups

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1669] [Questions / Documentation]
Hi @rcorces,

Is it possible to perform a statistical test comparing two groups or clusters used for the TF footprinting comparison? (similar to what HINT-ATAC can achieve but applicable to ArchR). Basically asking whether the motif footprinting depth is significantly different between the two groups.

I'd appreciate your suggestions.

Thank you.

**Comment by rcorces** [comment]
No, we do not provide this. The footprinting done by ArchR is "meta" footprinting which aggregates over many different sites. This is very different than single-site footprinting which attempts to define the precise binding site of a single TF in a single peak. In my own personal opinion, statistical comparison of meta footprints is no different than looking for enrichment of motifs in peaks for example by the hypergeometric test, which ArchR does have and is described in the manual.
